src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +60,138 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
+
+* [GC.GetGCMemoryInfo().GenerationInfo[i].SizeAfterBytes](https://docs.microsoft.com/dotnet/api/system.gcgenerationinfo):
+  Represents the size in bytes of a generation on exit of the GC reported in GCMemoryInfo.
+
+* [GCGenerationInfo.FragmentationAfterBytes Property](https://docs.microsoft.com/dotnet/api/system.gcgenerationinfo.fragmentationafterbytes#system-gcgenerationinfo-fragmentationafterbytes)
+  Gets the fragmentation in bytes on exit from the reported collection.
+
+### JIT Compiler related metrics
+
+These metrics are only available when targeting .NET6 or later.
+
+| Name                                                  | Description                                                                                                                                                                                                                                                                                                                                  | Units       | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**jit.il_compiled.size**       | Count of bytes of intermediate language that have been compiled since the process start. The value will be zero under ahead-of-time (AOT) compilation mode.                                                                                                                                                                                  | `bytes`     | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**jit.methods_compiled.count** | The number of times the JIT compiler compiled a method since the process start. The JIT compiler may be invoked multiple times for the same method to compile with different generic parameters, or because tiered compilation requested different optimization settings. The value will be zero under ahead-of-time (AOT) compilation mode. | `{methods}` | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**jit.compilation_time**       | The amount of time the JIT compiler has spent compiling methods since the process start. The value will be zero under ahead-of-time (AOT) compilation mode.                                                                                                                                                                                  | `ns`        | ObservableCounter | `Int64`    |                  |                  |
```

Comment:
```suggestion
| process.runtime.dotnet.**jit.compilation_time**       | The amount of time the JIT compiler has spent compiling methods since the process start.                                                                                                                                                                               | `ns`        | ObservableCounter | `Int64`    |                  |                  |
```